Core/clim-core/frames.lisp

```diff
@@ -1668,13 +1670,14 @@ have a `pointer-documentation-pane' as pointer documentation,
 	      (when last-presentation
 		(do-hilite last-presentation (last-window) :unhighlight))
 	      (setq last-presentation presentation
-		    feedback-fn (feedback dest-translator)
-		    hilite-fn (highlighting dest-translator))
+		    feedback-fn       (when dest-translator (feedback dest-translator))
+		    hilite-fn         (when dest-translator (highlighting dest-translator)))
```

Comment:
I wonder if we should set it to NIL or we should use default functions as specified for `define-drag-and-drop-translator` (namely `frame-drag-and-drop-highlighting` and `frame-drag-and-drop-feedback`). If we do the latter, then this should be if and we don't have to check whenever feedback-fn is NIL.

If you happen to hit slot of the translator being NIL, I think this is a problem which should be handled during translator definition (so it defaults to what it is meant to, and if NIL is passed explicitly that should error).

```
 		(do-hilite last-presentation (last-window) :unhighlight))
-	      (setq last-presentation presentation
-		    feedback-fn (feedback dest-translator)
-		    hilite-fn (highlighting dest-translator))
+	      (if dest-translatorn
+                  (setq last-presentation presentation
+                        feedback-fn (feedback dest-translator)
+                        hilite-fn (highlighting dest-translator))
+                  (setq last-presentation presentation
+                        feedback-fn #'frame-drag-and-drop-feedback
+                        hilite-fn #'frame-drag-and-drop-highlighting))
```

Reply:
If there is no `dest-translator`, highlighting should happen, as far as I can tell. I adapted the suggestion accordingly.